pkg/kv/postgres/migrate.go

```diff
@@ -54,15 +51,15 @@ func Migrate(ctx context.Context, dbPool *pgxpool.Pool, dbParams params.Database
 	}
 	defer store.Close()
 
-	shouldDrop, err := getMigrationStatus(ctx, store)
+	shouldTruncate, err := getMigrationStatus(ctx, store)
```

Comment:
I know I said we should truncate - but we've decided to keep with drop, so maybe revert this?